src/main/kotlin/com/ajaxproject/warehouse/dto/WaybillCreateDto.kt

```diff
@@ -1,23 +1,21 @@
 package com.ajaxproject.warehouse.dto
 
 import jakarta.validation.Valid
-import jakarta.validation.constraints.Min
+import jakarta.validation.constraints.NotEmpty
 import jakarta.validation.constraints.NotNull
 import java.time.LocalDate
 
 data class WaybillCreateDto(
-    @field:NotNull(message = "customer id must be provided")
-    @field:Min(1, message = "id must be valid")
-    val customerId: Int?,
+    @field:NotEmpty(message = "customerId must be provided")
+    val customerId: String?,
     @field:NotNull(message = "date must be provided")
     val date: LocalDate?,
     @field:Valid
-    val products: List<WaybillProductCreateDto>? = null
+    val products: List<MongoWaybillProductCreateDto> = emptyList()
 ) {
-    data class WaybillProductCreateDto(
-        @field:NotNull(message = "product id must be provided")
-        @field:Min(1, message = "id must be valid")
-        val id: Int?,
+    data class MongoWaybillProductCreateDto(
```

Comment:
The goal of dto is to have abstraction (maybe from a database. I understand if you named entity MongoEntity, because it's related to mongo, but don't do this to dto